src/main/java/duke/TaskType.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+/**
+ * Enum representing the type of a task.
+ */
+public enum TaskType {
+    DEADLINE("D"),
+    EVENT("E"),
+    TODO("T");
+
+    private final String taskIcon;
+    private TaskType(String taskIcon) {
+        this.taskIcon = taskIcon;
+    }
+
+    /**
+     * Getter for task icon.
+     * @return Task icon corresponding to type of task.
+     */
```

Comment:
I'm not so sure if documentation for getters is necessary. To compare it with your `TaskList` class, there are trivial methods there that are undocumented. Hence, to be consistent, I suggest, either adding documentation to those methods as well, or removing documentation for methods whose purpose is clear from the name.

Reply:
I'll work on that in future iterations after I reorganise my code for the GUI, thanks for the suggestions!